lib/ex_force/client/tesla/tesla.ex

```diff
@@ -49,7 +62,7 @@ defmodule ExForce.Client.Tesla do
         {Tesla.Middleware.JSON, engine: Jason},
         {Tesla.Middleware.Headers, get_headers(opts)}
       ],
-      Keyword.get(opts, :adapter)
+      {config(:adapter), [name: config(:name)]}
```

Comment:
We are getting the adapter configurations from app `config` yet the opts being passed here is only specific to `finch`. We either support passing all adapters alongside their opts, or just use finch by default here without giving user any configurability capabilities